### ISSUE TITLE
Cannot add author to record 01GJGF9E2HBV6KH7GA6Y4410AB

### ISSUE THREAD

**Issue by mietcls** [ugent-library/biblio-backoffice#855]

**Comment by mietcls** [comment]
Note, could be related to case where we tried to add an author to 8770198 before?

![Image](https://user-images.githubusercontent.com/1572885/203823288-faf7404e-724a-4aa6-a5fb-c27619b19e0e.png)

**Comment by nics** [comment]
@netsensei  i would fix the data + degrade gracefully if the frontend api is down or the call fails

**Comment by netsensei** [comment]
@nics Just discussed this with @mietcls 

* Fixing the data in particular cases like this is the short & quick fix to this problem. 
* Adding department name to contributors isn't all that crucial. Being able to add an author really is. So, graceful degradation should be: ignore/skip if the call fails because the data isn't there.

Beyond that, we need graceful degradation / error handling for the entire integration with the frontend API. Which would be a separate story / epic.

**Comment by mietcls** [comment]
Sanity check for myself: we can see the ID of the department, just not the full name.

**Comment by mietcls** [comment]
@netsensei any updates? The reviewers need to help this person out by 6 December, and might need some time to review all of his records.

**Comment by netsensei** [comment]
@mietcls Yes. A fix has been rolled out. It's now possible to add the author. The department with the author record won't contain the department full name. Only the ID.

**Comment by mietcls** [comment]
Superb, thanks! Give me a shout when I can test.

**Comment by netsensei** [comment]
You can test now! :mega:

**Comment by mietcls** [comment]
Tested and confirmed by Marthe
